src/sentry/scim/endpoints/teams.py

```diff
@@ -118,6 +118,10 @@ def _add_members_operation(self, request, operation, team):
             member = OrganizationMember.objects.get(
                 organization=team.organization, id=member["value"]
             )
+            if OrganizationMemberTeam.objects.filter(team=team, organizationmember=member).exists():
+                # if a member already belongs to a team, do nothing
+                continue
+
```

Comment:
Shame this isn't combined with `OrganizationMemberTeam.objects.create`. Feels like it wants a "create if doesn't already exist" type method. I don't think the object manager has anything directly like that; it might be achievable by catching an exception but that probably isn't clean. Anyway, I don't think it would be worth tying ourselves into knots here, especially before opening the transaction -- just a thought.

Also, general nit: when I see `continue`, I check whether there is a way to factor it out. One way would be to extract the loop body into a helper function where the `continue` can become a `return`. It's fine here, but I'd want to revisit that if the method grows.

Reply:
agreed, some sort of upsert method would be nice. and yeah, the unfortunate thing about a helper function here is having to pass around all of those arguments here. I suppose we could make it an inner function but felt like a lot to avoid a `continue` hehe.